features/test/src/index.html

```diff
@@ -1,255 +1,246 @@
-        <button
-            id="comm.polyIn.match.get_array_with_single_quad_with_blank_node_subject"
-            onclick="testFeature.execute(() => testFeature.canGetArrayWithSingleQuadWithBlankNodeSubjectFromPolyInMatch())"
-        >
-            comm.polyIn.match.get_array_with_single_quad_with_blank_node_subject
-        </button>
-        <button
-            id="comm.polyIn.match.get_array_with_single_quad_with_named_node_object"
-            onclick="testFeature.execute(() => testFeature.canGetArrayWithSingleQuadWithNamedNodeObjectFromPolyInMatch())"
-        >
-            comm.polyIn.match.get_array_with_single_quad_with_named_node_object
-        </button>
-        <button
-            id="comm.polyIn.match.get_array_with_single_quad_with_blank_node_object"
-            onclick="testFeature.execute(() => testFeature.canGetArrayWithSingleQuadWithBlankNodeObjectFromPolyInMatch())"
-        >
-            comm.polyIn.match.get_array_with_single_quad_with_blank_node_object
-        </button>
-        <button
-            id="comm.polyIn.match.get_array_with_single_quad_with_literal_object"
-            onclick="testFeature.execute(() => testFeature.canGetArrayWithSingleQuadWithLiteralObjectFromPolyInMatch())"
-        >
-            comm.polyIn.match.get_array_with_single_quad_with_literal_object
-        </button>
-        <button
-            id="comm.polyIn.match.get_array_with_single_quad_with_named_node_graph"
-            onclick="testFeature.execute(() => testFeature.canGetArrayWithSingleQuadWithNamedNodeGraphFromPolyInMatch())"
-        >
-            comm.polyIn.match.get_array_with_single_quad_with_named_node_graph
-        </button>
-        <button
-            id="comm.polyIn.match.get_array_with_single_quad_with_blank_node_graph"
-            onclick="testFeature.execute(() => testFeature.canGetArrayWithSingleQuadWithBlankNodeGraphFromPolyInMatch())"
-        >
-            comm.polyIn.match.get_array_with_single_quad_with_blank_node_graph
-        </button>
-        <button
-            id="comm.polyIn.match.get_array_with_single_quad_with_default_graph"
-            onclick="testFeature.execute(() => testFeature.canGetArrayWithSingleQuadWithDefaultGraphFromPolyInMatch())"
-        >
-            comm.polyIn.match.get_array_with_single_quad_with_default_graph
-        </button>
-        <button
-            id="comm.polyIn.match.get_array_with_multiple_quads"
-            onclick="testFeature.execute(() => testFeature.canGetArrayWithMultipleQuadsFromPolyInMatch())"
-        >
-            comm.polyIn.match.get_array_with_multiple_quads
-        </button>
+        <h1>Status</h1>
+        <div>
+            <span id="status">Starting</span>
+            <span id="result"></span>
+        </div>
+
+        <h1>Input</h1>
+        <div>
+            <input id="input.1" />
+            <input id="input.2" />
+            <input id="input.3" />
+            <input id="input.4" />
+        </div>
+
+        <h1>Tests</h1>
```

Comment:
Not using poly-look components here? Two birds, one stone?

Reply:
OMG no, that's overkill :D I'm just trying to make this work again, not rewrite it.